pkg/controller/vitessshard/reconcile_backup_job.go

```diff
@@ -109,9 +110,34 @@ func (r *ReconcileVitessShard) reconcileBackupJob(ctx context.Context, vts *plan
 		vts.Status.HasInitialBackup = corev1.ConditionTrue
 	}
 
+	// Reconcile vtbackup PVCs. Use the same key as the corresponding Pod,
+	// but only if the Pod expects a PVC.
+	err := r.reconciler.ReconcileObjectSet(ctx, vts, pvcKeys, labels, reconciler.Strategy{
+		Kind: &corev1.PersistentVolumeClaim{},
+
+		New: func(key client.ObjectKey) runtime.Object {
+			return vttablet.NewPVC(key, specMap[key].TabletSpec)
+		},
+		PrepareForTurndown: func(key client.ObjectKey, obj runtime.Object) *planetscalev2.OrphanStatus {
+			// Same as reconcileTablets, keep PVCs of Pods in any Phase
+			pod := &corev1.Pod{}
+			if getErr := r.client.Get(ctx, key, pod); getErr == nil || !apierrors.IsNotFound(getErr) {
```

Comment:
This differs from coordinating over the orphanedPods map. I'm assuming that PVC reconciliation continues and this will eventually turndown after the Pod is deleted.